### PR TITLE
osde2e job configuration for addon-ocs-converged

### PR DESCRIPTION
**Pull request by brainfunked**

### CODE REVIEW

File: ci-operator/jobs/openshift/osde2e/openshift-osde2e-main-periodics.yaml [hunk 1]
```diff
@@ -154,6 +154,61 @@ periodics:
...
+      - $(CONFIGS)
+      command:
+      - /osde2e
+      env:
+      - name: ADDON_IDS
+        value: ocs-operator
+      - name: ADDON_TEST_HARNESSES
+        value: quay.io/mkarnikredhat/ocsci-osd
+      - name: SECRET_LOCATIONS
+        value: /usr/local/osde2e-common,/usr/local/osde2e-credentials
```
Comment (jeefy) [inline ci-operator/jobs/openshift/osde2e/openshift-osde2e-main-periodics.yaml:184]: You should not use the `osde2e-credentials` and instead you should use `/usr/local/ocs-convered-credentials`

File: ci-operator/jobs/openshift/osde2e/openshift-osde2e-main-periodics.yaml [hunk 2]
```diff
@@ -154,6 +154,61 @@ periodics:
...
+        readOnly: true
+      - mountPath: /usr/local/osde2e-credentials
+        name: osde2e-credentials
+        readOnly: true
+    serviceAccountName: ci-operator
+    volumes:
+    - name: osde2e-common
+      secret:
+        secretName: osde2e-common
+    - name: osde2e-credentials
```
Comment (jeefy) [inline ci-operator/jobs/openshift/osde2e/openshift-osde2e-main-periodics.yaml:209]: This name and the secret name should match the defined secrets below (`ocs-converged-credentials`)

**Review by jeefy — CHANGES_REQUESTED** [review]
Job definition looks pretty good! Only thing that needs updating is the secrets. Let me know if you have any questions about the requested changes.

**Comment by brainfunked** [comment]
@jeefy As we discussed on Slack, I'm closing this PR into two separate ones to get the secret mapping merged first.

**Pull request closed without merging**